app/src/main/java/eu/kanade/domain/track/interactor/AddTracks.kt

```diff
@@ -79,19 +79,19 @@ class AddTracks(
 
     suspend fun bindEnhancedTrackers(manga: Manga, source: Source) = withNonCancellableContext {
         withIOContext {
-            getTracks.await(manga.id)
```

Comment:
This only returned already active tracks, which I don't think is intended.

Reply:
Seems like the previous maintainer fucked up during a cleanup